src/constants/buyCover.ts

```diff
@@ -1,2 +1,17 @@
+export enum CoverAsset {
+  ETH = 0,
+  DAI = 1,
+}
+
+export enum CoverId {
+  BUY = 0,
+}
```

Comment:
Should these be in the `types/` folder? 🤔 Could be either I guess. Feedback please!

Reply:
enums are a set of named constants. enums are ok to be put in types dir only if there is no existing constants file already (my opinion, but happy to discuss further)

Reply:
Nah, this is small enough to not matter. We'll see when the project grows 👌